File: examples/quickstart/after_openai.py
from openai import OpenAI

with OpenAI(base_url="http://localhost:3000/openai/v1", api_key="not-used") as client:
    response = client.chat.completions.create(
        model="tensorzero::function_name::generate_haiku",
        messages=[
            {
                "role": "user",
                "content": "Write a haiku about TensorZero.",
            }
        ],
    )

print(response)

File: examples/quickstart/after_openai.ts
import OpenAI from "openai";

const client = new OpenAI({
  baseURL: "http://localhost:3000/openai/v1",
});

const response = await client.chat.completions.create({
  model: "tensorzero::function_name::generate_haiku",
  messages: [
    {
      role: "user",
      content: "Write a haiku about TensorZero.",
    },
  ],
});

console.log(JSON.stringify(response, null, 2));
